### PR TITLE
Add information about contributing to Windows images

### PR DESCRIPTION
**Pull request by vsafonkin**
Added more clarified information about contributing to Windows images

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -15,6 +15,12 @@ Please note that this project is released with a [Contributor Code of Conduct][c
 1. [Fork][fork] and clone the repository
 1. Create a new branch: `git checkout -b my-branch-name`
 1. Make your changes, ensure that they include steps to install and validate post-install (e.g. [kind.sh](images/linux/scripts/installers/kind.sh)).
+
+      Adding tool for Windows images:
+    - add tool installation script `Install-ToolName.ps1` (`/images/win/scripts/Installers`)
+    - add tool validation script `Validate-ToolName.ps1` (the same directory)
+    - add changes to document tool name and version to autogenerated readme file: `images/win/scripts/SoftwareReport/SoftwareReport.Generator.ps1`
```
Comment (AlenaSviridenko) [inline CONTRIBUTING.md:22]: may be rewrite to something like `document tool name and version in autogenerated README file <here goes the path to file>`

Comment (vsafonkin) [reply CONTRIBUTING.md]: @AlenaSviridenko , reworked, please review

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -15,6 +15,12 @@ Please note that this project is released with a [Contributor Code of Conduct][c
 1. [Fork][fork] and clone the repository
 1. Create a new branch: `git checkout -b my-branch-name`
 1. Make your changes, ensure that they include steps to install and validate post-install (e.g. [kind.sh](images/linux/scripts/installers/kind.sh)).
+
+      Adding tool for Windows images:
+    - add tool installation script `Install-ToolName.ps1` (`/images/win/scripts/Installers`)
+    - add tool validation script `Validate-ToolName.ps1` (the same directory)
+    - add changes to document tool name and version to software report generator: `images/win/scripts/SoftwareReport/SoftwareReport.Generator.ps1` (it uses to generate software readme file, e.g. [Windows2019-Readme.md](images/win/Windows2019-Readme.md))
```
Comment (AlenaSviridenko) [inline CONTRIBUTING.md:22]: `it uses to generate software readme file` -> `it is used to generate software README file`

Comment (AlenaSviridenko) [reply CONTRIBUTING.md:22]: but still better to update text to something like

- document tool name and version to software report generator `images/win/scripts/SoftwareReport/SoftwareReport.Generator.ps1` that is used to generate software README file, e.g. [Windows2019-Readme.md](images/win/Windows2019-Readme.md)

Comment (vsafonkin) [reply CONTRIBUTING.md]: Fixed, thank you!

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -15,6 +15,12 @@ Please note that this project is released with a [Contributor Code of Conduct][c
 1. [Fork][fork] and clone the repository
 1. Create a new branch: `git checkout -b my-branch-name`
 1. Make your changes, ensure that they include steps to install and validate post-install (e.g. [kind.sh](images/linux/scripts/installers/kind.sh)).
+
+      Adding tool for Windows images:
+    - add tool installation script `Install-ToolName.ps1` (`/images/win/scripts/Installers`)
+    - add tool validation script `Validate-ToolName.ps1` (the same directory)
+    - add changes to document tool name and version to software report generator: `images/win/scripts/SoftwareReport/SoftwareReport.Generator.ps1` that is used to generate software README file, e.g. [Windows2019-Readme.md](images/win/Windows2019-Readme.md)
```
Comment (miketimofeev) [inline CONTRIBUTING.md:22]: **the** software report generator

Comment (vsafonkin) [reply CONTRIBUTING.md]: Fixed, thanks!

**Pull request merged**